test/iron-overlay-behavior.html

```diff
@@ -141,6 +141,24 @@
           });
         });
 
+        test('close an overlay in proximity to another overlay', function(done) {
+          var secondOverlay = fixture('basic');
+          // Open and close a separate overlay.
+          secondOverlay.open();
+          secondOverlay.close();
+
+          // Open the overlay we care about.
+          overlay.open();
```

Comment:
would this work if you moved the open to below the `addEventListener`, so that the "immediately" comment looked more immediate?